app/assets/stylesheets/widgets/single_column/messaging.scss

```diff
@@ -75,100 +146,135 @@
 	}
 
 	@include media($medium-screen) {
+
 		.text-column {
 			margin: 0 auto;
 			width: 75%;
 		}
 	}
 	@include media($large-screen) {
+
 		.text-column {
 			width: 60%;
 		}
 	}
-
 }
 
 /* Two Column Variation */
 .messaging-widget__2-column {
 	background-color: #eaeaea;
+
 	/* All Variations */
 	.media-column {
 		white-space: nowrap;
 		margin-bottom: $base-spacing * 2;
 		padding: 0;
 	}
+
 	.text-column {
 		text-align: left;
 	}
 
+  .actions {
```

Comment:
@kaylaziegler I renamed "actions" to "text-actions". Since this is an existing style class/container, pages that use the 1-Column Modular templates will need republished out. I republished the three example pages I sent previously, but I did not want to kick off a full site republish without your permission.